spec/lib/local-links-manager/service_spec.rb

```diff
@@ -0,0 +1,25 @@
+describe "service" do
+  context ":duplicate" do
```

Comment:
```suggestion
describe "Service tasks" do
  describe "service:duplicate" do
```

This file also seems to be in an unconventional place, a more conventional one would be spec/lib/tasks/service_spec.rb